pulsar-broker/src/test/java/org/apache/pulsar/client/impl/RawReaderTest.java

```diff
@@ -84,6 +83,7 @@ private Set<String> publishMessages(String topic, int count, boolean batching) t
             .enableBatching(batching)
             // easier to create enough batches with a small batch size
             .batchingMaxMessages(10)
+            .batchingMaxPublishDelay(1, TimeUnit.MINUTES)
```

Comment:
We should flush the batch manually.
This method is used in multiple places.
If the count is not (n * 10), the test will always wait for 1 minute.

Reply:
It is make sense, we can flush manually after sent the last message.